package.json

```diff
@@ -40,21 +40,21 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "react-feather": "2.0.10",
-    "sanitize-html": "2.16.0",
-    "swagger-ui-react": "5.21.0"
+    "sanitize-html": "2.17.0",
+    "swagger-ui-react": "5.24.1"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "3.7.0",
+    "@docusaurus/module-type-aliases": "3.8.1",
     "@tsconfig/docusaurus": "2.0.3",
     "@types/prismjs": "1.26.5",
-    "@types/react": "18.3.5",
+    "@types/react": "18.3.9",
     "@types/react-router-dom": "5.3.3",
-    "@types/sanitize-html": "2.15.0",
-    "@typescript-eslint/eslint-plugin": "8.31.1",
-    "@typescript-eslint/parser": "8.31.1",
+    "@types/sanitize-html": "2.16.0",
+    "@typescript-eslint/eslint-plugin": "8.34.0",
+    "@typescript-eslint/parser": "8.34.0",
     "docusaurus-plugin-module-alias": "0.0.2",
-    "eslint": "8.57.0",
-    "eslint-config-prettier": "10.1.2",
+    "eslint": "9.29.0",
```

Comment:
Looks like there's some migration needed as part of the eslint 9 update as the [lint step](https://github.com/openfga/openfga.dev/actions/runs/15830414169/job/44621429772?pr=1054) in CI is now failing, I think we need to migrate the existing [`.eslintrc.json`](https://github.com/openfga/openfga.dev/blob/d64dd1cc197134fd4880eac555620b6d885d1cc1/.eslintrc.json) to [eslint.config.js](https://eslint.org/docs/latest/use/migrate-to-9.0.0#flat-config)

```
ESLint: 9.29.0

ESLint couldn't find an eslint.config.(js|mjs|cjs) file.
```